test/parallel/test-repl-eval.js

```diff
@@ -10,15 +10,16 @@ const repl = require('repl');
     eval: common.mustCall((cmd, context) => {
       // Assertions here will not cause the test to exit with an error code
       // so set a boolean that is checked in process.on('exit',...) instead.
-      evalCalledWithExpectedArgs = (cmd === 'foo\n' && context.foo === 'bar');
+      evalCalledWithExpectedArgs = (cmd === 'function f() {}' &&
+                                    context.foo === 'bar');
     })
   };
 
   const r = repl.start(options);
   r.context = {foo: 'bar'};
 
   try {
-    r.write('foo\n');
+    r.write('function f() {}\n');
```

Comment:
Can I suggest that you add a comment describing why this particular input was chosen?